experiments/backtest.py

```diff
@@ -46,9 +46,8 @@ def run_backtest(
 ) -> tuple[pd.Series, pd.DataFrame]:
     """
     Run a simplified backtest for a given strategy.
-    At time t we use data from t-500 to t-1 to forecast the data and
-    compute the optimal portfolio weights and cash holdings.
-    We then trade to these weights at time t.
+    At time t we use data from t-lookback to t to compute the optimal portfolio
+    weights and then execute the trades at time t.
```

Comment:
If we literally include the returns of `t+1` in our forecast, I think the additional complexity of observing prices at `t-1`, then creating the optimal weights and trading at `t` did not make sense. The below changes are a result of this.